app/mailers/problem_order_mailer.rb

```diff
@@ -1,23 +1,23 @@
 # frozen_string_literal: true
 
 class ProblemOrderMailer < ApplicationMailer
-
   def notify_user(order_detail)
     @order_detail = order_detail
     @user = @order_detail.user
-    mail(to: @user.email, subject: text("notify_user.subject", facility: @order_detail.facility.abbreviation))
+    reply_to = @order_detail.facility.email || SettingsHelper.setting("email.from")
+    mail(to: @user.email, reply_to: reply_to, subject: text("notify_user.subject", facility: @order_detail.facility.abbreviation))
   end
 
   def notify_user_with_resolution_option(order_detail)
     @order_detail = order_detail
     @user = @order_detail.user
-    mail(to: @user.email, subject: text("notify_user.subject", facility: @order_detail.facility.abbreviation))
+    reply_to = @order_detail.facility.email || SettingsHelper.setting("email.from")
+    mail(to: @user.email, reply_to: reply_to, subject: text("notify_user.subject", facility: @order_detail.facility.abbreviation))
   end
 
   protected
 
   def translation_scope
     "views.problem_order_mailer"
   end
-
```

Comment:
Please put the newlines back in.  Maybe you still need to update your VSCode settings?